Bundle/PageBundle/Twig/Extension/PageExtension.php

```diff
@@ -153,6 +165,25 @@ public function cmsPageBusinessPagePatternSiteMap(BasePage $page)
     }
 
     /**
+     * Construct CSS link markup for the style of all the Widgets contained in the current View.
+     *
+     * @return string
+     */
+    public function cmsPageCss()
+    {
+        $currentView = $this->currentViewHelper->getCurrentView();
+
+        if(!$currentView) {
+            return '';
+        }
+
+        return sprintf(
+            '<link ng-href="%s" rel="stylesheet" type="text/css" rel="stylesheet"/>',
```

Comment:
Try without angular to avoid to wait for javascript to be load before loading css